winsup/cygwin/scripts/gendef

```diff
@@ -117,13 +117,15 @@ EOF
 	.global $fe
```

Comment:
Can the TODO few lines above be already reverted?

Reply:
Yes, or it can be moved below the implementation.

Reply:
So far it looks like $fe wrapper is ported.